### DISCUSSION TITLE
Load Test on Oracle Sharded Database

### DISCUSSION THREAD

**Discussion by sunilbhu** [TPC-Council/HammerDB#150] [Q&A] [answered]
Is HammerDB can be used to load test Oracle Sharded database 19c ?

**Comment by sm-shaw** [comment]
The Oracle Sharded Database has not been tested with either of the HammerDB workloads based on TPC-C or TPC-H.  having said that TPC-H does lend itself to partioning so if interested in a workload that can be sharded this is where I would recommend starting.

**Comment by sm-shaw** [comment]
Closed no further update

**Comment by sunilbhu** [comment]
Do you mean HammerDB can use Sharded tables ?

**Comment by sm-shaw** [comment, marked as answer]
As previous guidance HammerDB has not been tested against a sharded database. However HammerDB is completely open and both schmea builds and drivers are scripted for the reason that you are able to modify them exactly as you wish, in this case to run against a sharded database.
